assets/src/edit-story/components/canvas/layout.js

```diff
@@ -132,28 +132,29 @@ const PageAreaSafeZone = styled.div`
   overflow: visible;
   position: relative;
   margin: auto 0;
-`;
 
-const PageAreaDangerZone = styled.div`
-  pointer-events: none;
+  ${({ showSafeZone }) =>
+    showSafeZone &&
+    `&::before {
+  content: '';
+  width: 20px;
+  height: var(--page-height-px);
```

Comment:
if you want better syntax highlighting and formatting from your editor you can import `css` from styled components and do:
```
  ${({ showSafeZone }) =>
    showSafeZone && css`
      &::before {
        content: '';
        width: 20px;
        height: var(--page-height-px);
```

Have asked Phil about this before and he said `css` is just an internal utility and there should be no perf cost of using it so it's totally cool to use.

Reply:
Ahh dope that was driving me crazy.  Making this change now.